### PR TITLE
Add info on how DS/NetComm safety controls work

### LINKED ISSUE #2227: Add info on how DS/NetComm safety controls work
Copy from: https://www.chiefdelphi.com/t/severe-100-packet-loss/428031/139?u=peter_johnson

(without the potential fixes piece, which is my opinion)

### PR DESCRIPTION
**Pull request by pjbuterbaugh**
Closes #2227.

### CODE REVIEW

**Comment by pjbuterbaugh** [comment]
TODO: Add updated figures for DS logging/console.

File: source/docs/hardware/hardware-safety/placeholder.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Placeholder for Safety Writeup
+==============================
+
+.. Note:: From a Chief Delphi `thread reply <https://www.chiefdelphi.com/t/severe-100-packet-loss/428031/139?u=peter_johnson/>`_ by WPILib Developer Peter Johnson.
```
Comment (sciencewhiz) [inline source/docs/hardware/hardware-safety/placeholder.rst:4]: This should be in the PR description/commit message, but doesn't need to be in the article

File: source/docs/hardware/hardware-safety/placeholder.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Control \(DS->Robot\) and status \(Robot->DS\) packets have an embedded sequence number. The DS uses these to compute round-trip-time and packet loss. A status packet that\’s returned is marked as “lost” if the RTT is greater than ~250 ms. This does not mean it was actually missing \(no response received\). The DS does keep a separate count of truly missing \(e.g. no response\) packets and disables \(starts sending control packets with enable=false\) after ~10 drops occur \(so I think this works out to ~450 ms, assuming it\’s 250+10*20\).
+- High CPU / GUI delays result in the DS continuing to send packets with enable=true for a period of time until that loop is notified a disable occurs.
+
+Vendor Components:
+^^^^^^^^^^^^^^^^^^
+
+- CTRE uses a custom approach that reads the disable indicator on NetComm and stops motors within 100 `ms` of a disable.
+
+Potential Improvements:
```
Comment (sciencewhiz) [inline source/docs/hardware/hardware-safety/placeholder.rst:33]: I would delete this section as it isn't really useful to teams in understanding how the current system works

File: source/docs/software/frc-glossary.rst [hunk 3]
```diff
@@ -131,6 +137,9 @@ FRC Glossary
    pose
       The collection of position and rotation information that describes how a rigid body is oriented in space, relative to some fixed reference point.
 
+   PWM
```
Comment (sciencewhiz) [inline source/docs/software/frc-glossary.rst:140]: I would link to the FRC docs pages rather then wikipedia

**Comment by pjbuterbaugh** [comment]
PR is connected to the wrong branch; closing and opening a new draft PR to continue discussion.

**Pull request closed without merging**